mappings/net/minecraft/entity/EntityType.mapping

```diff
@@ -31,15 +31,15 @@ CLASS net/minecraft/class_1299 net/minecraft/entity/EntityType
 		ARG 3 feetY
 		ARG 5 feetZ
 	METHOD method_17684 fromNbt (Lnet/minecraft/class_2487;)Ljava/util/Optional;
-		ARG 0 compoundTag
+		ARG 0 nbt
 	METHOD method_17685 getWidth ()F
 	METHOD method_17686 getHeight ()F
 	METHOD method_17842 loadEntityWithPassengers (Lnet/minecraft/class_2487;Lnet/minecraft/class_1937;Ljava/util/function/Function;)Lnet/minecraft/class_1297;
-		ARG 0 compoundTag
+		ARG 0 nbt
 		ARG 1 world
 		ARG 2 entityProcessor
 	METHOD method_17848 loadEntityFromNbt (Lnet/minecraft/class_2487;Lnet/minecraft/class_1937;)Ljava/util/Optional;
```

Comment:
Should this be `fromNbt`?

Reply:
MMmh, I'm not sure, this call `EntityType#getEntityFromNbt` surrounded with a try/catch and an error logging then a empty return in the catch block.

A pure `from` doesn't usually return an Optional either.